contrib/rpm/README.md

```diff
@@ -19,7 +19,7 @@ reference the release tarball available from https://digibyte.org/bin/ and
 `Source1` should reference the BerkeleyDB source.
 
 Sources 10-99 are for source files that are maintained in the
-[DigiByte git repository](https://github.com/digibyte/digibyte) but are not part of
+[DigiByte git repository](https://github.com/DigiByte-Core/digibyte) but are not part of
```

Comment:
Should probs even just look at removing this file / directory, given we don't maintain any rpms and this is out of date